src/pages/Privacy/PopUp.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Link from '../../components/common/Link';
+import Button from '@material-ui/core/Button';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import { CardHeader } from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '360px',
+    height: '206px',
+    zIndex: 50000,
+    position: 'fixed',
+    right: '1.51%',
+    bottom: '0.5%',
+  },
+  title: {
+    fontSize: '20px',
+    fontWeight: '700',
+  },
+  headStyle: {
+    padding: 8,
+  },
+  iconStyle: {
+    verticalAlign: 'bottom',
+  },
+  copyStyle: {
+    paddingTop: 5,
+    width: '308px',
+  },
+  hoverContent: {
+    fontSize: 12,
+    textAlign: 'justify',
+    lineHeight: '16.2px',
+  },
+  linkStyle: {
+    padding: 0,
+    color: '#FFB100',
+    marginLeft: 3,
+    textDecoration: 'none',
+  },
+  imageStyle: {
+    width: '26px',
+    height: '26px',
+    marginRight: '2%',
+    verticalAlign: 'middle',
+  },
+  buttonStyle: {
+    width: '150px',
+    height: '36px',
+    color: '#0F1D2F',
+    fontWeight: '700',
+  },
+  linkText: {
+    color: '#0F1D2F',
+    fontWeight: '700',
+    paddingRight: '25px',
+  },
+}));
+
+const PopUp = () => {
+  const classes = useStyles();
+  const date = new Date();
+  const setCookie = function (cookieEmail, cookieValue, expDays) {
+    date.setTime(date.getTime() + expDays);
+
+    const expires = 'expires=' + date.toUTCString();
+
+    document.cookie = cookieEmail + cookieValue + ';' + expires + ';path=/';
+
+    return document.cookie;
+  };
+
+  const deleteCookie = function (cookieEmail) {
+    date.setTime(date.getTime());
+    const expires = 'expires=' + date.toUTCString();
+    document.cookie = cookieEmail + expires + ';path=/';
+  };
+  const acceptCookieConsent = () => {
+    deleteCookie('civictecindex_cookie_consent');
+    setCookie('civictecindex_cookie_consent', 1, 30);
```

Comment:
Since cookieValue and expDays are predefined, we can declare them as variables inside setCookie and change them whenever.
```suggestion
  const setCookie = function (cookieEmail) {
    const cookieValue = 1;
    const expDays = 30;
    date.setTime(date.getTime() + expDays);

    const expires = 'expires=' + date.toUTCString();

    document.cookie = cookieEmail + cookieValue + ';' + expires + ';path=/';

    return document.cookie;
  };

  const deleteCookie = function (cookieEmail) {
    date.setTime(date.getTime());
    const expires = 'expires=' + date.toUTCString();
    document.cookie = cookieEmail + expires + ';path=/';
  };
  const acceptCookieConsent = () => {
    deleteCookie('civictecindex_cookie_consent');
    setCookie('civictecindex_cookie_consent');
```

Reply:
> So, I'm not sure what the intended behavior for the cookie should be: currently the way it looks like it's working is the date is registered upon the user visiting the site, not when the user clicks 'Accept' on the cookie pop-up.
> 
> One concern I wanted to bring up is in certain scenarios where the user takes a while to select 'Accept', or visits the page but leaves it open for a long period of time.
> 
> This could potentially paint an inaccurate picture of traffic. Then again, maybe we only want to know the time when they just visit the site.
> 
> Wanted to bring this up to attention after playing around with and testing the cookie in the browser. :)

Hi Dennis
I was not told anything about functionality, I was just told to code as per reference with 311 data. I think the functionality will be told after